### DISCUSSION TITLE
Housing/case for top-hat rail (ebus adapter)

### DISCUSSION THREAD

**Discussion by XueSheng-GIT** [john30/ebusd#642] [General]
I'm thinking about getting an ebus adapter v3 with ethernet, but couldn't find any info if there's a housing/case available (preferably for a top-hat rail installation within the electrical cabinet).

Where do you you typically put this ebus adapter? Wouldn't be the electrical cabinet the typical place to mount such an adapter!?

**Comment by john30** [comment]
see [here for adapter and housing](https://github.com/eBUS/adapter/issues/9) discussions
